test/gtest/ucp/test_ucp_request.cc

```diff
@@ -398,8 +418,8 @@ UCS_TEST_P(test_proto_reset, tag_eager_multi_bcopy, "ZCOPY_THRESH=inf",
 UCS_TEST_P(test_proto_reset, get_offload_bcopy_to_get_am_bcopy,
            "ZCOPY_THRESH=inf", "RNDV_THRESH=inf")
 {
-    if (has_transport("ib") && !has_resource(sender(), "rc_mlx5")) {
-        UCS_TEST_SKIP_R("No IB resources");
+    if (has_shm_rma()) {
+        UCS_TEST_SKIP_R("SHM was selected for RMA operations");
```

Comment:
maybe add helper function like skip_no_pending_rma() for lines 421-423 (and in other test functions) ?